### PR TITLE
Adding in Danger Systems to verify changelog and readme, enable support for future rules

### PR DESCRIPTION
**Pull request by rshade**
### Description
Adding in Danger Systems to verify changelog and readme, enable support for future rules. I.e version present, versions match,etc. 

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Comment by rshade** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: target of repeat operator is not specified: /*.pt/
```
 #  from Dangerfile:1
 #  -------------------------------------------
 >  has_app_changes = !git.modified_files.grep(/*.pt/).empty?
 #  # Changelog entries are required for changes to library files.
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+cache:
+  directories:
+    - bundle
+
+rvm:
+  - 2.4.1
+
+bundler_args: "--without documentation --path bundle"
+
+before_install:
```
Comment (crunis) [inline .travis.yml]: Why do you do this? doesn't it happen automatically by default? Just curious

Comment (rshade) [reply .travis.yml]: I stole the .travis.yml from the Danger repo. I dont know :)

Comment (crunis) [reply .travis.yml]: No problem, just curious, looks totally good to merge to me

**Pull request merged**